libpod/runtime_ctr.go

```diff
@@ -38,6 +38,7 @@ func (r *Runtime) NewContainer(rSpec *spec.Spec, options ...CtrCreateOption) (c
 		return nil, err
 	}
 	ctr.config.StopTimeout = CtrRemoveTimeout
+	ctr.config.ShmSize = DefaultShmSize
```

Comment:
This is already set in newContainer in container.go

Reply:
Removed